src/openms/include/OpenMS/CHEMISTRY/IsotopeDistribution.h

```diff
@@ -224,6 +239,14 @@ namespace OpenMS
     /// convolves the distribution @p input with itself and stores the result in @p result
     void convolveSquare_(ContainerType & result, const ContainerType & input) const;
 
+    /** @brief calculates the fragment distribution for a fragment molecule and stores it in @p result.
+
+        @param fragment_isotope_dist the isotopic distribution of the fragment (as if it was a precursor).
+        @param comp_fragment_isotope_dist the isotopic distribution of the complementary fragment (as if it was a precursor).
+        @param precursor_isotopes a list of which precursor isotopes were isolated
+     */
+    void calcFragmentIsotopeDist_(ContainerType& result, const ContainerType& fragment_isotope_dist, const ContainerType& comp_fragment_isotope_dist, const std::vector<UInt>& precursor_isotopes);
```

Comment:
I will let you make the call on this. The other functions in this file were all out1, in1, in2, ... so I kept it consistent with what was around it. I can definitely flip them around though if you want to start enforcing that convention.